plugins/backend-trace-acceptor/src/Cardano/BM/Backend/TraceAcceptor.lhs

```diff
@@ -140,8 +140,8 @@ mkAcceptor (RemoteSocket host port) = handleError TraceAcceptorSocketError $ mdo
 acceptConnection :: FromJSON a => Trace.Trace IO a -> Socket.Socket -> IO ()
 acceptConnection baseTrace  sock = handleError TraceAcceptorServerError $ do
   (client, _sa) <- Socket.accept sock
-  handle <- Socket.socketToHandle client IO.ReadWriteMode
-  _client <- Async.async $ clientThread baseTrace handle
+  h <- Socket.socketToHandle client IO.ReadWriteMode
+  _client <- Async.async $ clientThread baseTrace h
```

Comment:
IMHO, `handle` is better name than `h`.

Reply:
the compiler warned that "handle" is already imported and this name would hide that function. so even if this name is quite good it conflicts with another in scope.